src/qt/locale/bitcoin_sv.ts

```diff
@@ -2687,8 +2687,8 @@ Var vänlig och försök igen.</translation>
         <translation>Skicka trace-/debuginformation till terminalen istället för till debug.log</translation>
     </message>
     <message>
-        <source>Send transactions as zero-fee transactions if possible (default: %u)</source>
-        <translation>Sänd transaktioner som nollavgiftstransaktioner om möjligt (förvalt: %u)</translation>
+        <source>Send as zero-fee transaction (default: %u)</source>
```

Comment:
remove all of these "(default: %u)"

%u will crash the client because it will be expecting another parameter to be put in the "sprintf" for this text, but currently the client doesn't do that as the default text is "Send as zero-fee transaction"

Reply:
Im on it, thanks!